Source/Core/DolphinQt/Debugger/MemoryWidget.h

```diff
@@ -57,6 +57,7 @@ class MemoryWidget : public QDockWidget
   void OnDumpFakeVMEM();
 
   std::vector<u8> GetValueData() const;
+  u32 GetTargetAddress(bool* good_address, bool* good_offset) const;
```

Comment:
Any reason this doesn't return a struct like:

```cpp
struct SomeName
{
  u32 address = 0;
  bool is_good_address = false;
  bool is_good_offset = false;
};
```
?

This would get rid of the out parameters and allow structured bindings.

It also gets rid of the uninitialized reads that are present in the existing code from forgetting to initialize the bools before calling the function (since `|=` is used in the function)

Reply:
The struct sounds good to me.

Pedantic: good_address and good_offset are initialized by QString's toUInt and toShort functions before the `|=`s, but the fact that isn't obvious is a point in favor of the struct.